test/e2e/edu-helpers.js

```diff
@@ -64,16 +63,27 @@ function completeRelativeInformation(client, data, onlyRequiredFields) {
     .clearValue('input[name="root_relativeSocialSecurityNumber"]')
     .setValue('input[name="root_relativeSocialSecurityNumber"]', data.relativeSocialSecurityNumber)
     .clearValue('input[name="root_relativeDateOfBirthYear"]')
-    .setValue('input[name="root_relativeDateOfBirthYear"]', parseInt(dobFields[0], 10).toString())
-    .click(relationshipInput);
+    .setValue('input[name="root_relativeDateOfBirthYear"]', parseInt(dobFields[0], 10).toString());
   selectDropdown(client, 'root_relativeDateOfBirthMonth', parseInt(dobFields[1], 10).toString());
   selectDropdown(client, 'root_relativeDateOfBirthDay', parseInt(dobFields[2], 10).toString());
 
+  if (typeof data.relationship !== 'undefined') {
+    const relationshipInput = data.relationship ? `input[value="${data.relationship}"]` : 'input[name="root_relationship_1"]';
```

Comment:
Is this just in case `data.relationship` is somehow `null`, `''`, or `false`? It just seems like the check for undefined _should_ cover it.